site/content/en/docs/formats/mpii.md

```diff
@@ -0,0 +1,88 @@
+---
+title: 'MPII Human Pose Dataset'
+linkTitle: 'MPII Human Pose Dataset'
+description: ''
+weight: 1
+---
+
+## Format specification
+
+The original MPII Human Pose Dataset is available
+[here](http://human-pose.mpi-inf.mpg.de/#overview).
+
+Supported annotation types:
+- `Bbox`
+- `Points`
+
+Supported attributes:
+- `center` (list)
+- `scale` (float)
+
+## Import MPII Human Pose Dataset
+
+Datumaro does not support `MATLAB` annotation files. Instead,
+`JSON` and `NUMPY` files are supported.
+
+A Datumaro project with a MPII Human Pose Dataset source can be
+created in the following way:
+
+```bash
+datum create
+datum import --format mpii <path/to/dataset>
+```
+
+It is also possible to import the dataset using Python API:
+
+```python
+from datumaro.components.dataset import Dataset
+
+mpii_dataset = Dataset.import_from('<path/to/dataset>', 'mpii')
+```
+
+MPII Human Pose Dataset directory should have the following structure:
+
+<!--lint disable fenced-code-flag-->
+```
+dataset/
+├── jnt_visible.npy # optional
```

Comment:
Will this format support metafile?

Reply:
We can extend the `dataset meta file` for `PointsCategories`. What do you think about it?

Reply:
We don't own this format, you better ask AC if they see it is useful.

Reply:
In AC, only these categories are used. I think this will be enough for Datumaro too.